<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MCSE</class>
 <widget class="QMainWindow" name="MCSE">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>180</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Georgia</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Minecraft Sounds Extractor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="mcse_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>240</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Minecraft Sounds Extractor</string>
    </property>
   </widget>
   <widget class="QLabel" name="mc_dir_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Minecraft directory:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mc_dir_lineEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>530</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="mc_dir_pushButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>40</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="out_dir_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Output directory</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="out_dir_lineEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>530</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="out_dir_pushButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>80</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="extract_pushButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>120</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Extract</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ver_comboBox">
    <property name="geometry">
     <rect>
      <x>939</x>
      <y>40</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ver_label">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>40</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Version:</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
